lib/client/chart.js

```diff
@@ -358,6 +384,12 @@ function init (client, d3, $) {
           .attr('transform', 'translate(0,' + chartHeight + ')')
           .call(chart.xAxis2);
 
+        var basalsTransition = chart.basals.transition();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['basalsTransition' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1129533692/issues/source?bid=2452379&fileBranchId=2886508#l387)